### PR TITLE
updated yaml anchors in additional dependencies

### PR DESCRIPTION
**Pull request by Kunsh-Manghwani**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -39,10 +44,7 @@ repos:
     rev: 3.9.2
     hooks:
     -   id: flake8
-        additional_dependencies:
-            - flake8-comprehensions==3.1.0
-            - flake8-bugbear==21.3.2
-            - pandas-dev-flaker==0.2.0
+        additional_dependencies: *version_check_dependencies
```
Comment (MarcoGorelli) [inline .pre-commit-config.yaml:47]: what about the `yesqa` hook?

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

**Comment by Kunsh-Manghwani** [comment]
yes sir i just created created yaml anchor for yesqa hook

File: .pre-commit-config.yaml (hunk 1 above)
Comment (Kunsh-Manghwani) [reply .pre-commit-config.yaml:47]: sorry sir , i just made a commit resolving yesqa hook

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
this isn't working for me:
```console
$ pre-commit run flake8 --files pandas/core/generic.py 
An error has occurred: InvalidConfigError: 
==> File .pre-commit-config.yaml
=====> while parsing a block mapping
  in "<unicode string>", line 1, column 1
did not find expected key
  in "<unicode string>", line 11, column 1
Check the log at /home/marco/.cache/pre-commit/pre-commit.log
```

Can you try running the above?

**Comment by Kunsh-Manghwani** [comment]
Sir , i am not able to understand what i have to change . The anchor is applied in both the hooks and and code has passed all 5 checks successfully.

**Comment by Kunsh-Manghwani** [comment]
Please guide me i am new here

**Comment by Kunsh-Manghwani** [comment]
pre-commit is not working in my system is there any other way to test these files

**Comment by MarcoGorelli** [comment]
> pre-commit is not working in my system 

what do you mean? can you show some output?

**Comment by Kunsh-Manghwani** [comment]
image
pre-commit is installed but it is still showing this error:

'pre-commit' is not recognized as an internal or external command,
operable program or batch file.

**Comment by MarcoGorelli** [comment]
please see [here](https://pandas.pydata.org/pandas-docs/stable/development/contributing_environment.html) for how to set up a development environment - once you have it set up and have pre-commit working, you can try this again

> pre-commit is not working in my system is there any other way to test these files

**Pull request closed without merging**